### PR TITLE
migrate NLP examples to INC new API

### PR DESCRIPTION
**Pull request by violetch24**
## Type of Change

migrate NLP examples to new API
text-classification, question-answering, language-modeling

## Description

JIRA ticket: https://jira.devtools.intel.com/browse/ILITV-2625

### CODE REVIEW

File: examples/pytorch/nlp/huggingface_models/language-modeling/quantization/ptq_static/fx/run_benchmark.sh [hunk 1]
```diff
@@ -57,9 +57,9 @@ function run_benchmark {
     extra_cmd=''
 
     if [[ ${mode} == "accuracy" ]]; then
-        mode_cmd=" --accuracy_only "
-    elif [[ ${mode} == "performance" ]]; then
-        mode_cmd=" --benchmark "
+        mode_cmd=" --accuracy "
+    elif [[ ${mode} == "benchmark" ]]; then
```
Comment (chensuyue) [inline examples/pytorch/nlp/huggingface_models/language-modeling/quantization/ptq_static/fx/run_benchmark.sh:61]: The input mode is "performance".

File: examples/pytorch/nlp/huggingface_models/language-modeling/quantization/ptq_static/fx/run_tuning.sh [hunk 2]
```diff
@@ -44,13 +44,28 @@ function run_tuning {
 
     if [ "${topology}" = "gpt_j_wikitext" ]; then
         TASK_NAME='wikitext'
-        model_name_or_path=$input_model 
-        extra_cmd='--dataset_config_name=wikitext-2-raw-v1' 
+        model_name_or_path=${input_model}
+        model_type='gpt'
```
Comment (chensuyue) [inline examples/pytorch/nlp/huggingface_models/language-modeling/quantization/ptq_static/fx/run_tuning.sh:48]: Where do we use this `model_type`?

File: examples/pytorch/nlp/huggingface_models/language-modeling/quantization/ptq_static/fx/README.md [hunk 3]
```diff
@@ -30,9 +29,16 @@ python run_clm.py \
   --tune \
   --output_dir /path/to/checkpoint/dir
 ```
+## Language-Modeling Task
```
Comment (chensuyue) [inline examples/pytorch/nlp/huggingface_models/language-modeling/quantization/ptq_static/fx/README.md:32]: Let's remove this `run_tuning.sh` method, it's duplicate with `python run_clm.py`. It doesn't make sense for users without model download path.

File: examples/pytorch/nlp/huggingface_models/language-modeling/quantization/ptq_static/fx/run_benchmark.sh (hunk 1 above)
Comment (violetch24) [reply examples/pytorch/nlp/huggingface_models/language-modeling/quantization/ptq_static/fx/run_benchmark.sh]: fixed it.

File: examples/pytorch/nlp/huggingface_models/language-modeling/quantization/ptq_static/fx/run_tuning.sh (hunk 2 above)
Comment (violetch24) [reply examples/pytorch/nlp/huggingface_models/language-modeling/quantization/ptq_static/fx/run_tuning.sh]: deleted it.

**Pull request closed without merging**